schemas/app-interface/app-interface-settings-1.yml

```diff
@@ -52,6 +52,12 @@ properties:
         type: string
       credentials:
         "$ref": "/common-1.json#/definitions/vaultSecret"
+  imap:
+    type: object
+    additionalProperties: false
+    properties:
```

Comment:
since you made `credentials` required in the graphql schema, you can should add a `required` section here as well